local-dev/sql_validate.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+SERVICE=$1
+
+# validate mysql
+echo "sleeping for 5 seconds during postgres boot..."
+sleep 5
+export PGPASSWORD=${SERVICE}-test
+psql --username ${SERVICE}-test --no-password --host=postgres --port=5432 -d testdb -c "SELECT VERSION();SELECT NOW()"
```

Comment:
why doesn't this need a password? maybe get rid of the SERVICE=$1 since it's always the janitor?

Reply:
I think this is no password promot. 
https://www.postgresql.org/docs/12/app-psql.html
The actually password is set by the PGPASSWORD env variable